app/javascript/pages/map/recent-imagery/components/recent-imagery-settings/recent-imagery-settings-styles.scss

```diff
@@ -0,0 +1,114 @@
+@import '~styles/settings.scss';
+
+.c-recent-imagery-settings {
+  position: absolute;
+  width: rem(600px);
+  padding: rem(25px) 0 rem(22px) 0;
+  transform: translateX(-50%);
+  background-color: $white;
+  box-shadow: 0 1px 3px 0 rgba($black, 0.25);
+  z-index: 102;
+
+  .dragger-icon {
+    position: absolute;
+    top: rem(9px);
+    left: rem(10px);
+    width: rem(6px);
+    height: rem(10px);
+    fill: $slate;
+  }
+
+  .close-btn {
+    position: absolute;
+    top: 0;
+    right: 0;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: rem(30px);
+    height: rem(30px);
+    cursor: pointer;
+
+    svg {
+      width: rem(9px);
+      height: rem(10px);
+      fill: $slate;
+    }
+  }
+
+  &__title {
```

Comment:
Maybe it is a good time to start using css modules? Then we dont have to worry about these things